notifier/bot/sampleData.py

```diff
@@ -0,0 +1,37 @@
+# ! shall be replaced by django dynamic models
+courses = [
+    {
+      "term_code": "202210",
+      "crn": "11243",
+      "course_number": "ACCT110",
+      "department_code": "ACFN",
+      "section_number": "01",
+      "course_title": "Introduction to Financial Accounting",
+      "available_seats": 0,
+      "waiting_list_count": 1,
+      "class_type": "LEC",
+      "class_days": "UT",
+      "start_time": "1100",
+      "end_time": "1215",
+      "building": "76",
+      "room": "2233",
+      "instructor_name": " "
+    },
+    {
+      "term_code": "202210",
+      "crn": "11245",
+      "course_number": "ACCT110",
+      "department_code": "ACFN",
+      "section_number": "03",
+      "course_title": "Introduction to Financial Accounting",
+      "available_seats": 1,
+      "waiting_list_count": 5,
+      "class_type": "LEC",
+      "class_days": "MW",
+      "start_time": "1100",
+      "end_time": "1215",
+      "building": "24",
+      "room": "158",
+      "instructor_name": " "
+    }
+]
```

Comment:
u should not push a mock data to the repo, either add its folder in `.gitignore` or don't include it